includes/Modules/Site_Verification.php

```diff
@@ -96,6 +101,12 @@ function () {
 			}
 		);
 
+		add_action(
+			'googlesitekit_invalidate_verification_meta_cache',
+			function () {
+				$this->options->delete( self::OPTION_VERIFICATION_META_TAGS );
```

Comment:
I'm not really fond of changing this to an option. Yes, for most people it will result in the same, but transients use the object cache if available, and furthermore using a transient clearly shows the intent of using this as a cache, which here no longer is obvious (the source of truth for this data still is elsewhere).

I'm not sure it's worth passing in the `Transients` instance to each module just for this, but what we should do is
* either instantiate a `Transients` class property just for this one module class
* or just use the plain transients API

I think we can go with the `Transients` instantiation.

Reply:
I opted for using plain transients for now as we don't yet have any module classes instantiating other classes internally that they don't receive as constructor dependencies. This wouldn't be hard to change to use `Transients` instead though. If you'd prefer to go this route, just let me know how you would like to use it - instantiate in the one method or override the module's constructor and instantiate an instance to an instance property on the class?

Reply:
I think because it's only used there, we could instantiate `Transients` in the method for a once-off object. I think using that class is a bit more future-proof because of the "never-existing" network mode :)